stories/Checkbox.stories.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+import { Checkbox as Component  } from "../confetti-ds/src";
+
+export default {
+  title: "Checkbox/Checkbox",
+  component: Component,
+  argTypes: {
+    value: {
+      control: {
+        type: "select",
+        options: ["string", "number", "bool"]
```

Comment:
same this as the `radio` PR. remove this control, since it doesn't make much sense.

Reply:
fixed